server/server.go

```diff
@@ -4223,3 +4223,29 @@ func (s *Server) changeRateLimitLogInterval(d time.Duration) {
 	default:
 	}
 }
+
+// Disconnects a client by cid
+func (s *Server) DisconnectClientByID(id uint64) error {
+	client := s.clients[id]
+	if client != nil {
+		client.closeConnection(Kicked)
+		return nil
+	}
+	return errors.New("no such client id")
+}
+
+func (s *Server) LDMClientByID(id uint64) error {
+	info := s.copyInfo()
+	info.LameDuckMode = true
+
+	c := s.clients[id]
+	if c != nil && c.opts.Protocol >= ClientProtoInfo &&
+		c.flags.isSet(firstPongSent) {
+		// sendInfo takes care of checking if the connection is still
+		// valid or not, so don't duplicate tests here.
+		s.Debugf("sending LDM info to cid=%d", id)
```

Comment:
Can we use the client logger instead here maybe? That would include the cid in the logger fields already.
```suggestion
		c.Debugf("sending lame duck mode INFO to client")
```

Reply:
Done